sys/net/routing/rpl/rpl.c

```diff
@@ -116,6 +110,11 @@ uint8_t rpl_init(int if_id, ipv6_addr_t *address)
     return SIXLOWERROR_SUCCESS;
 }
 
+kernel_pid_t rpl_get_if(void)
```

Comment:
@authmillenon Since this is still the old ip/sixlo stack there is only this *global independent thread approach* without any ties to interfaces. Am I wrong?